_config/theme.yml

```diff
@@ -0,0 +1,9 @@
+
+SilverStripe\LoginForms\EnablerExtension:
+  login_themes:
+    - "silverstripe/login-forms:accessway"
+    - $default
+  excluded_actions: ['index', 'logout', 'ping']
```

Comment:
Why are we excluding logout? You have a form to confirm you want to log out if you hit the URL directly:

![image](https://user-images.githubusercontent.com/5170590/56010208-dbdfb880-5d36-11e9-99e6-2e7d075e1fed.png)

Reply:
I'll remove it from the list.